### PR TITLE
Updates the PDA painter so that it paints standard PDAs. Restores PDA painter functionality after the PDA GAGS PR.

### LINKED ISSUE #58842: PDAs can't be painted to "standard" aka assistant's PDAs
## Round ID:
N/A

## Testmerges:
N/A

## Reproduction:
Insert PDA to PDA and ID painter and choose standard PDA. Remove PDA from machine and PDA is still the same as before.

### PR DESCRIPTION
**Pull request by Greniza**
## About The Pull Request

Closes [58842](https://github.com/tgstation/tgstation/issues/58842#issue-873702774)

Also fixes the PDA painter after [58844](https://github.com/tgstation/tgstation/pull/58844) updated the way the PDA was displayed without updating the PDA painter.

## Why It's Good For The Game

Minor bug-fix.
Restores functionality to the PDA painter after 58844.

## Changelog
:cl:
fix: Made the PDA painter be able to paint PDAs into standard PDAs.
fix: Made the PDA painter able to paint PDAs again after 58844 did not update the painter.
/:cl:

celotajstg did much of the writing in the final product. He deserves a majority of the GBP.

### CODE REVIEW

**Comment by Greniza** [comment]
Of note: the failstate for improperly implementing a new PDA's interaction within `SSid_access.station_pda_templates` is moved from the PDA not being repainted to the PDA being painted to a standard PDA.

**Comment by celotajstg** [comment]
waiting for https://github.com/Greniza/tgstation/pull/5 to get merged for a working fix

**Comment by celotajstg** [comment]
Good to merge

File: code/game/machinery/PDApainter.dm [hunk 1]
```diff
@@ -314,13 +314,22 @@
...
-				if(!(pda_types[path] == selection))
-					continue
+				if(pda_types[path] == selection)
+					pda_path = path
+					break
+
+			if(initial(pda_path.greyscale_config) && initial(pda_path.greyscale_colors))
+				stored_pda.greyscale_config = initial(pda_path.greyscale_config)
+				stored_pda.greyscale_colors = initial(pda_path.greyscale_colors)
+				stored_pda.update_greyscale()
```
Comment (ninjanomnom) [inline code/game/machinery/PDApainter.dm:327]:
```suggestion
				stored_pda.set_greyscale_config(initial(pda_path.greyscale_config), update=FALSE)
				stored_pda.set_greyscale_colors(initial(pda_path.greyscale_colors))
```

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Pull request merged**